jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -310,4 +348,35 @@ private boolean isWrapper(String valueClass) {
 
         return wsEndpoint;
     }
+
+    /**
+     * Check if a URI string contains any sequence with //, /./, or /../
+     *
+     * @param uriString
+     * @return
```

Comment:
specification comment doesn't seem to be consistent with the method signature

Reply:
What do you mean? Checking if a URI has relative paths involves checking for `//`, `/./`, and `/../` sequences.